src/main/java/callhub/connect/drivers/MessageWebSocketAccess.java

```diff
@@ -1,24 +1,43 @@
 package callhub.connect.drivers;
+import com.google.gson.Gson;
 import org.springframework.messaging.handler.annotation.DestinationVariable;
 import org.springframework.messaging.handler.annotation.MessageMapping;
 import org.springframework.messaging.handler.annotation.SendTo;
 import org.springframework.stereotype.Controller;
 
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.Locale;
+
 @Controller
 public class MessageWebSocketAccess {
 
+    private static final Gson gson = new Gson();
+    final static DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("HH:mm:ss a", Locale.CANADA);
+
     @MessageMapping("/message-customer/{sessionId}")
     @SendTo("/topic/message-employee/{sessionId}")
     public String sendMessageCustomer(@DestinationVariable String sessionId, String message) throws Exception {
-        System.out.println(message);
-        return message + " (" + sessionId + ")";
+        HashMap<String, String> response = generateResponse(message);
+        return gson.toJson(response);
     }
 
     @MessageMapping("/message-employee/{sessionId}")
     @SendTo("/topic/message-customer/{sessionId}")
     public String sendMessageEmployee(@DestinationVariable String sessionId, String message) throws Exception {
-        System.out.println(message);
-        return message + " (" + sessionId + ")";
+        HashMap<String, String> response = generateResponse(message);
+        return gson.toJson(response);
+    }
+
+    private HashMap<String, String> generateResponse(String message){
```

Comment:
i would move this to a dataaccessobject or a use case tbh

Reply:
done